issuer-web/src/router/guards/pres-req.ts

```diff
@@ -0,0 +1,31 @@
+import IssuerStore from "@/store";
+import { Route } from "vue-router";
+import { decode, verify } from "jsonwebtoken";
+import { AppConfig } from '@/models/appConfig';
+import * as ConfigService from "@/services/config";
+
+
+
+export default async function hasPresReq(
+  to: Route,
+  from: Route,
+  next: Function
+) {
+  const config: any = await ConfigService.getAppConfig();
```

Comment:
Please use types where possible: in this case `const config: AppConfig`. It will help catch errors at compilation time (thanks Typescript!) and it will enable autocompletion in the editor when accessing object properties.